docs/getting-started/ios-native.mdx

```diff
@@ -89,7 +89,7 @@ This is done to overcome a bug with Xcode 11 which fails to compile swift code w
 Install the dependencies by running `pod install`. You can now import and initialize Flipper in your
 AppDelegate.
 
-<Tabs defaultValue="ios" values={[{ label: 'iOS', value: 'ios'}, { label: 'Swift', value: 'swift'}]}>
+<Tabs defaultValue="ios" values={[{ label: 'ObjC', value: 'ObjC'}, { label: 'Swift', value: 'swift'}]}>
```

Comment:
Should we also change `defaultValue`?

Reply:
CC @tbskgm0

Reply:
Yes, that's right.
It's strange that the defaultValue is iOS when the values are objC and swift.
In the following docs, the defaultValue is objC. So, I think the defaultValue should be set to objC again.
[<Tabs defaultValue="objc" values={[{ label: 'ObjC', value: 'objc'}, { label: 'Swift', value: 'swift'}]}>](https://github.com/facebook/flipper/blob/main/desktop/plugins/public/network/docs/setup.mdx)

![](https://user-images.githubusercontent.com/89886782/154062381-68c973e9-8109-4bf4-8481-45b852518cd0.png)